### DISCUSSION TITLE
Indirect expand in view collection?

### DISCUSSION THREAD

**Discussion by Xyphyn** [pocketbase/pocketbase#2016] [Q&A]
Hello, I have two collections: `communities` and `communityCounts`. `communityCounts` is a view collection that stores stats about communities (# of members, posts, etc, so that I don't have to get a bunch of useless data from an expansion.). I was wondering if I could get a list of communities, and for each community expand communityCounts?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand your collections configuration. Does `communityCounts` has a `relation` field to the community?

In any case, if you haven't already, for indirect expand you can check https://pocketbase.io/docs/expanding-relations/#indirect-expand.

If the above doesn't help, please provide more details about your collections configuration (_you can export your collections from "Admin UI > Settings > Export collections"_).

**Reply by Xyphyn** [reply]
It's a view collection, so it's not technically a relation as i'm understanding it. Is there any way to indirect expand based on its ID?

**Reply by Xyphyn** [reply]
I don't know how I could relate to a single community.

**Comment by Kworz** [comment]
I think i have the same behavior as @Xyphyn, indirect expanding from a view collection seems to not work.

I have one base collection (article) that is defined by :
- id
- price
- quantity

and one view collection (article_view) defined as follows:
```sql
SELECT id, id as article, price, quantity, price*quantity as stock_price from article
```

When indirect expanding the relation using `article_view(article)`, the expand field stays empty no matter what i try. I've set API rules to be allowing everything.

If it's normal behavior, i will this down, otherwise i'll create an issue :)

**Reply by Xyphyn** [reply]
The issue I think is that you don't actually have a relation for your view collection. This was fixed for me in #2029

**Reply by Kworz** [reply]
Well using this technique, by writing
```sql
SELECT article.id, article.id as article, article.price*article.quantity as stock_price FROM (select 1) as dummy,  article
```
it works, thank you ! :)

**Reply by Xyphyn** [reply]
The real person to thank is ganigeorgiev :)
